### DISCUSSION TITLE
Where is my Quotations

### DISCUSSION THREAD

**Discussion by precizelabs-admin** [Erratums/ims-support#12] [Q&A] [answered]
I can't find quotations in invoice history

**Comment by erratumsolutions** [comment, marked as answer]
We have a separate module for Questions. You can see this in the Modules list. 

To make it easier, customers are allowed to create quotations from Invoice itself by selecting **Invoice** **Type** as **Quotation** as below 

![](https://s3-ap-south-1.amazonaws.com/ind-cdn.freshdesk.com/data/helpdesk/attachments/production/82012862796/original/sAASRjXq_4R8xu0Mb_AANUqN9gcyoGJ1WA.png?1629655263)

but the **Quotations** will be moved automatically into Module **Quotations.** You can access this from Modules as below 

[![](https://ims.esapps.in/knowledgebase/images/module_quotation_marking.png)](https://ims.esapps.in/knowledgebase/images/module_quotation_marking.png)

  
\
**Note:** You cannot add new Quotation from this Module. You will have to add it from Invoice itself.
